doc/modules/tree.rst

```diff
@@ -579,11 +579,17 @@ Note that it fits much slower than the MSE criterion.
 Missing Values Support
 ======================
 
-:class:`DecisionTreeClassifier` and :class:`DecisionTreeRegressor`
-have built-in support for missing values when `splitter='best'` and criterion is
+:class:`DecisionTreeClassifier`, :class:`DecisionTreeRegressor`
+have built-in support for missing values when `splitter='best'`, while
```

Comment:
Don't both of these have support for best and random splitters now?

Reply:
I think DTC is using greedy splits, so splitter='best', and ETC is using random splits, so splitter='random'.

Technically the trees support both arguments, but they would only ever be used in the context of a forest.

Reply:
See: [fea6cbf](https://github.com/scikit-learn/scikit-learn/pull/27966/commits/fea6cbf42534a7553644e4d06117d561924e6eeb)